test/server/admin/stats_handler_test.cc

```diff
@@ -36,6 +36,22 @@ class StatsHandlerTest {
     tls_.shutdownThread();
   }
 
+  void setHistogramBucketSettings() {
+    // Limit amount of buckets for tests.
+    envoy::config::metrics::v3::StatsConfig config;
+    auto& bucket_settings = *config.mutable_histogram_bucket_settings();
+
+    envoy::config::metrics::v3::HistogramBucketSettings setting;
+    setting.mutable_match()->set_prefix("h");
```

Comment:
Why `"h"`? Maybe comment on the significance of this value.

Reply:
I changed the setHistogramBucketsSettings implementation a little and added some comments in relevant places.

Reply:
That makes it much clearer. Thanks!